pkg/container/docker/kill.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
this file is not actually new, it was removed due to being dead code and brought back now so we can use it for haproxy. I'll probably make this a node level API instead in a follow-up cleanup.